src/Umbraco.Cms.Integrations.DAM.Aprimo/App_Plugins/UmbracoCms.Integrations/DAM/Aprimo/css/aprimo.css

```diff
@@ -0,0 +1,25 @@
+﻿.aprimo-editor-container {
```

Comment:
It's perhaps best to not provide this, such that the implementer can style how they want.  As it stands any custom styles would conflict with once we've provided here.